bricks/next-builder/src/data-providers/UpdateBricksToV3.ts

```diff
@@ -1,46 +1,136 @@
-import { InstanceApi_mixUpdateInstance } from "@next-sdk/cmdb-sdk";
-import { BackupBricks } from "./BackupBricks";
-import { NodeDetail, UpdateRouteOrTemplate } from "./UpdateBricks";
-import { createProviderClass } from "@next-core/brick-utils";
-
-export async function UpdateBrickToV3({
-  appId,
-  rootId,
-}: {
-  appId: string;
-  rootId: string;
-}): Promise<boolean> {
-  const result = await BackupBricks({
-    appId,
-    rootId,
-  });
-
-  if (result) {
-    const { rootNode, newWrapperBrick } = result;
-
-    const list = await UpdateRouteOrTemplate(
-      rootNode as NodeDetail,
-      newWrapperBrick.id,
-      {
-        updateProxy: true,
-        updateUseResolves: true,
-        updateUseBrickTransform: true,
-        updateChildContext: true,
-      }
-    );
+import { brickMap } from "./brickMap";
+import { omit, isEmpty } from "lodash";
+import walk from "../utils/walk";
+import { pipes } from "@next-core/pipes";
+import { NodeDetail } from "./UpdateBricksArrange";
+
+export function UpdateBricksToV3(graphData: pipes.GraphData): NodeDetail[] {
+  const list: Array<NodeDetail> = [];
+  const mountPointMap = new Map<string, string>();
+  function update(route: Array<NodeDetail>): NodeDetail[] {
+    route.forEach((node) => {
+      const matchV3Brick: Record<string, any> = brickMap[node.brick];
```

Comment:
使用 `hasOwnProperty` 判断，避免 `{}.toString` 或其他类似的原型链上的碰撞，下面有很多地方需要一样的处理

```suggestion
      const matchV3Brick: Record<string, any> = hasOwnProperty(brickMap, node.brick) ? brickMap[node.brick] : null;
```